tests/examples/migrations/dollar_brace_embedded_variable.php.in

```diff
@@ -0,0 +1,27 @@
+<?hh // strict
+
+/**
+ * Copyright (c) 2016, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional
+ * grant of patent rights can be found in the PATENTS file in the same
+ * directory.
```

Comment:
Use the same MIT header as the other files in this diff.

HHAST shouldn't have any BSD+PATENTS any more - but I see there's a few, I'll fix up the current ones separately.